### PR TITLE
Add context to title

### PR DESCRIPTION
**Pull request by bradystroud**
This rule was showing up as "Do you know the best way to get started?" on google 
I didn't know what it was about

### CODE REVIEW

File: rules/clean-architecture-the-best-way-to-get-started/rule.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 type: rule
 archivedreason: 
-title: Do you know the best way to get started?
+title: Do you know the best way to get started with Clean Architecture?
```
Comment (wicksipedia) [inline rules/clean-architecture-the-best-way-to-get-started/rule.md:4]:
```suggestion
title: Clean Architecture - Do you know the best way to get started?
```

**Review by wicksipedia — CHANGES_REQUESTED** [review]
alternate way to add context

**Review by adamcogan — APPROVED** [review]

**Comment by wicksipedia** [comment]
can we resolve the outstanding comment first?
not sure why @adamcogan approved it without adding his 2c on that

**Comment by adamcogan** [comment]
CC @pierssinclairssw 
I prefer the prefix "Clean Architecture -" 
As per https://www.ssw.com.au/rules/prefixes

**Comment by wicksipedia** [comment]
Committed Adam's preferred way

**Comment by bradystroud** [comment]
Thanks @wicksipedia - I still need your approval to merge 🙃

**Review by wicksipedia — APPROVED** [review]

**Pull request merged**